### PR TITLE
add: Basic md files and ESLint set up completed

### PR DESCRIPTION
**Pull request by Sonakshi1901**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+[![Starware](https://img.shields.io/badge/Starware-⭐-black?labelColor=f9b00d)](https://github.com/zepfietje/starware)
+
+<!-- [![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url] -->
+
+<br />
+<p align="center">
+  <a href="https://github.com/dscnitrourkela/project-oregano">
```
Comment (abhibhaw) [inline README.md:10]: Never copy paste aankh band krke 🤣

**Review by abhibhaw — CHANGES_REQUESTED** [review]
Readme needs a lot of changes!!

File: README.md (hunk 1 above)
Comment (Sonakshi1901) [reply README.md]: Thanks for pointing that out, bhaiya! I did not notice it then. The changes are made.

**Comment by riteshsp2000** [comment]
Hey @Sonakshi1901 quite a few changes required, ill list them all here.
- The title in README, is HackNITR, we need to be Frameboi.
- Update all the links at the starting part.
- In the table of contents, create a section for HackNITR and add the about project content right up there.
- And in the about project section describe this project and not hacknitr :|
- Built with, only the technologies are mentioned, add the links to all their websites as well
---

- Add keywords and proper description in package.json
- Not all essential files/folders are included in gitignore. use [gitignore.io](https://gitignore.io) generate them and use it for eslint and prettier as well. A few keywords for generating: gatsby node firebase yarn (add any extra if needed)
---

- We will be using tailwing and styled components, set those up as well (using twin macro and all)
- Checkout the gatsby config page, quite a few things are mentioned there, so make sure you set up most of those things and if need be others as well.

**Comment by Sonakshi1901** [comment]
Ok, got it.

**Comment by Sonakshi1901** [comment]
@riteshsp2000 Do we use the HackNITR logo or some other picture?

**Comment by Sonakshi1901** [comment]
@riteshsp2000 Do let me know if I have to make any more changes in the gatsby-config file.

File: README.md [hunk 2]
```diff
@@ -1 +1,190 @@
...
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+      </ul>
+        <li><a href="#built-with">Built With</a></li>
+        <li><a href="#hackNITR">HackNITR 3.0</a></li>
```
Comment (abhibhaw) [inline README.md:37]: This href is wrong

File: README.md [hunk 3]
```diff
@@ -1 +1,190 @@
...
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#contribution-guidelines">Contribution guidelines</a></li>
```
Comment (abhibhaw) [inline README.md:43]: this too wrong

**Review by abhibhaw — COMMENTED** [review]
repo cover is also missing

File: README.md (hunk 3 above)
Comment (Sonakshi1901) [reply README.md:43]: Is it contribution guidelines?

**Comment by Sonakshi1901** [comment]
> repo cover is also missing

@abhibhaw repo cover is in the images folder.

File: README.md (hunk 2 above)
Comment (Sonakshi1901) [reply README.md:37]: Ok

File: README.md [hunk 4]
```diff
@@ -1 +1,188 @@
...
+
+1.  **Fork** the repo on GitHub
+2.  **Clone** the project to your local system
+3.  **Commit** changes to your own separate branch
+4.  **Push** your work back up to your fork
+5.  Submit a **Pull request** so that we can review your changes
+
+NOTE 1: Please abide by the [Contributing Guidelines](https://github.com/dscnitrourkela/project-guava-web/blob/master/CONTRIBUTING.md).
+
+NOTE 2: Please abide by the [Code of Conduct](https://github.com/dscnitrourkela/project-guava-web/blob/master/CODE_OF_CONDUCT.md).
```
Comment (DesignrKnight) [inline README.md:144]: Links need correction

File: README.md [hunk 5]
```diff
@@ -1 +1,188 @@
...
+
+### Running the project.
+
+The project uses Yarn and not NPM. It is strictly advised to stick with Yarn so as to avoid dependency conflicts down the line.
+
+```
+## Checkout into the project client directory
```
Comment (DesignrKnight) [inline README.md:151]: Double check if this step is needed.

Comment (Sonakshi1901) [reply README.md:151]: Ok

File: README.md (hunk 4 above)
Comment (Sonakshi1901) [reply README.md:144]: Ok

**Review by riteshsp2000 — APPROVED** [review]

**Pull request merged**